### PR TITLE
Shortcode instructions

### PR DESCRIPTION
**Pull request by megarocks**
https://github.com/Abolitionist-Project/QuantiModo-WordPress-Plugin/issues/105

### CODE REVIEW

File: css/qmwp.css [hunk 1]
```diff
@@ -595,4 +595,24 @@ body.qmwp-lightbox-visible {
...
+    padding: 10px 0;
+    border-width: 1px 0;
+    border-color: #000;
+    border-style: solid;
+}
+
+table.available-shortcodes tr:last-of-type td {
+    border: none;
+}
+
```
Comment (mikepsinn) [inline css/qmwp.css]: Codacy Issue found: [Element (td.shortcode) is overqualified, just use .shortcode without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1044229173/issues/source?bid=2836188&fileBranchId=2881965#l615)

Comment (mikepsinn) [inline css/qmwp.css]: Codacy Issue found: [Element (td.shortcode) is overqualified, just use .shortcode without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1044404539/issues/source?bid=2836188&fileBranchId=2881965#l615)

Comment (mikepsinn) [inline css/qmwp.css]: Codacy Issue found: [Element (td.shortcode) is overqualified, just use .shortcode without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1044519405/issues/source?bid=2836188&fileBranchId=2881965#l615)

Comment (mikepsinn) [inline css/qmwp.css]: Codacy Issue found: [Element (td.shortcode) is overqualified, just use .shortcode without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1045029029/issues/source?bid=2836188&fileBranchId=2881965#l615)

Comment (mikepsinn) [inline css/qmwp.css]: Codacy Issue found: [Element (td.shortcode) is overqualified, just use .shortcode without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1045104169/issues/source?bid=2836188&fileBranchId=2881965#l615)

**Comment by megarocks** [comment]
https://github.com/Abolitionist-Project/QuantiModo-WordPress-Plugin/issues/110

**Comment by megarocks** [comment]
https://github.com/Abolitionist-Project/QuantiModo-WordPress-Plugin/issues/107

**Comment by megarocks** [comment]
https://github.com/Abolitionist-Project/QuantiModo-WordPress-Plugin/issues/105

File: QMWP.php [hunk 2]
```diff
@@ -1355,21 +1354,27 @@ function qmwp_bargraph_scatterplot_timeline($attributes)
...
+        } else if ($attributes['show_predictors_or_outcomes'] == 'predictors') {
+            $variableAs = 'effect';
+        }
 
         if (!is_null($variable)) {
             $pluginContentHTML = $this->set_js_variables($pluginContentHTML,
                 array(
                     'qmwpShortCodeDefinedVariable' => $variable,
-                    'qmwpShortCodeDefinedVariableAs' => $attributes['variable_as']
+                    'qmwpShortCodeDefinedVariableAs' => $variableAs
```
Comment (mikepsinn) [inline QMWP.php]: I think we need to rename these to $showPredictorsOrOutcomes

File: css/qmwp.css (hunk 1 above)
Comment (mikepsinn) [inline css/qmwp.css]: Codacy Issue found: [Element (td.shortcode) is overqualified, just use .shortcode without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1048970776/issues/source?bid=2836188&fileBranchId=2881965#l615)

**Pull request merged**